app/include_upgrade_mainnet.go

```diff
@@ -0,0 +1,20 @@
+//go:build mainnet
+
+package app
+
+import (
+	"github.com/babylonlabs-io/babylon/app/upgrades"
+	v1 "github.com/babylonlabs-io/babylon/app/upgrades/v1"
+	"github.com/babylonlabs-io/babylon/app/upgrades/v1/mainnet"
+)
+
+// init is used to include v1 upgrade for mainnet data
+func init() {
+	Upgrades = []upgrades.Upgrade{v1.CreateUpgrade(v1.UpgradeDataString{
```

Comment:
only place beside tests that imports mainnet and is included only if there is a `mainnet` in build tags
> Makefile sets by default this build tag